test/web-platform-tests/to-run.yaml

```diff
@@ -152,11 +150,11 @@ NodeIterator-removal.html: [fail, Unknown]
 DIR: domparsing
 
 DOMParser-parseFromString-html.html: [fail, needs to get "the active document's URL" which is not possible with one DOMParser shared across all windows]
-DOMParser-parseFromString-xml-doctype.html: [fail, Unknown]
+DOMParser-parseFromString-xml-doctype.html: [fail, saxes doesn't support  DTD validation]
 DOMParser-parseFromString-xml.html: [fail, needs to get "the active document's URL" which is not possible with one DOMParser shared across all windows]
-createContextualFragment.html: [fail, Unknown]
+createContextualFragment.html: [fail, Range is not implemented]
 innerhtml-05.xhtml: [fail, Unknown]
-insert_adjacent_html-xhtml.xhtml: [fail, Unknown]
+insert_adjacent_html-xhtml.xhtml: [fail, XML fragment parsing doesn't enforce well-formedness checks]
```

Comment:
That seems like a saxes bug; can you file it and link to it here?

Reply:
This issue has been fixed with saxes version [`3.1.6`](https://github.com/lddubeau/saxes/blob/master/CHANGELOG.md#316-2019-01-17). I took the opportunity to the dependency to the latest version.